Source/WebCore/editing/cocoa/HTMLConverter.mm

```diff
@@ -2442,6 +2443,29 @@ AttributedString attributedString(const SimpleRange& range)
     return HTMLConverter { range }.convert();
 }
 
+#if ENABLE(WRITING_TOOLS)
+static bool hasBlockquoteOrPreAncestor(const Node& node)
```

Comment:
Nit: doesn't seem like a bad method to have unconditional of `ENABLE(WRITING_TOOLS)`, either 😛